### PR TITLE
Add Training render

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,10 +3,12 @@
...
     "require": {
         "php": "^7.1",
-        "ext-pdo_sqlite": "*",
+
+        "ext-pdo": "*",
+        "ext-pdo_mysql": "*",
         "ext-intl": "*",
         "doctrine/doctrine-bundle": "^1.9",
-        "doctrine/orm": "^2.6",
+        "doctrine/orm": "2.7.x-dev",
```
Comment (TomasVotruba) [inline composer.json:11]: This is needed due to ClassLoader bug in 2.6

File: config/packages/easy_admin.yaml [hunk 2]
```diff
@@ -1,5 +1,18 @@
 easy_admin:
     entities:
-        - App\Entity\Place
-        - App\Entity\Training
-        - App\Entity\Trainer
+        Training:
+            class: 'App\Entity\Training'
+            form:
+                exclude_fields: ['trainingReferences', 'trainingTerms']
```
Comment (TomasVotruba) [inline config/packages/easy_admin.yaml:6]: Custom solution :roll_eyes:  :wink: 

See https://github.com/TomasVotruba/open-training/pull/7/files#diff-318660bf4cd1ad8a5d0e608e94df8fae for more

**Comment by TomasVotruba** [comment]
@JanMikes I'm mergin this so other PRs can target `master` branch, but your feedback will still help me here.

File: config/packages/easy_admin.yaml (hunk 2 above)
Comment (TomasVotruba) [reply config/packages/easy_admin.yaml:6]: Catchy post version ↓ :boom: 

https://www.tomasvotruba.cz/blog/2018/08/20/painful-experience-over-solutions-extend-configuratin-in-easy-admin-bundle-with-collector/

**Comment by JanMikes** [comment]
@TomasVotruba im finally on it. Month later 😿 hopefully it will be helpful 😄

File: README.md [hunk 3]
```diff
@@ -1,23 +1,34 @@
...
 
 ```bash
-git remote add origin ...
-git pull
+git clone ...
+cd open-trainings
 composer update
+```
+
+Then rename `.env.dist` to `.env` and complete variables:
```
Comment (JanMikes) [inline README.md:24]: Rename is not enough, copying is the way - Instruction should be `cp .env.dist .env` and edit it

File: config/packages/easy_admin.yaml (hunk 2 above)
Comment (JanMikes) [reply config/packages/easy_admin.yaml:6]: This is interesting! Thank you for sharing :+1:

**Review by JanMikes — COMMENTED** [review]
Hi, i like the changes.

Overall i was not sure what to focus on, i am not a pro symfony4 (and easy admin bundle) user yet, so for me it was more likely inspiration what and how can it be done.

What i like is dividing into packages and separating it as components, but there is way too much configuration and i will want to take closer look at it, because i like the idea.

What i dont like very much is the setters in entities, you know, i would go in more DDD way 😄 but i completely understand as this is a MVP and needs to be prototyped fast - i am almost sure you would not be able to use EasyAdminBundle with DDD way entities, so it is completely fine.

Thank you for sharing this with me, it was really inspiring!

**Pull request merged**